README.md

```diff
@@ -47,146 +35,37 @@ This package implements the slave side of the simple key-value protocol used  fo
-* [mbed-drivers](https://github.com/ARMmbed/mbed-drivers/pull/165)
-* [sal](https://github.com/ARMmbed/sal/pull/46)
-* [sockets](https://github.com/ARMmbed/sockets/pull/64)
-* [utest](https://github.com/ARMmbed/utest/pull/6)
-
-## Greentea test tools
-
-```Greentea``` is a test automation tool written in Python and designed to automate test execution for [mbed](www.mbed.com) projects encapsulated as [yotta](http://yottadocs.mbed.com) modules. Its key features include: integration with yotta module, test automation for yotta module's tests and reporting.
-
 ## Compatibility
 
 ```greentea-client``` is compatible with:
-* [Greentea](https://github.com/ARMmbed/greentea) v0.2.x onwards
-* [htrun](https://github.com/ARMmbed/htrun) v0.2.x onwards
-* [utest](https://github.com/ARMmbed/utest) v1.10.0 onwards
-* [unity](https://github.com/ARMmbed/utest) v2.0.1 onwards
-
-### Greentea support
-
-If you wish to use ```greentea-client``` please make sure you are using latest ```Greentea``` tools from [PyPI](https://pypi.python.org/pypi/mbed-greentea) (you will need Python 2.7):
-```
-$ pip install mbed-greentea --upgrade
-```
-
-*Note*: If you previously used ```mbed-drivers/test_env.h``` features to write your test cases please downgrade ```Greentea``` to a version lower than ```v0.2.0``` to stay compatible:
-```
-$ pip install "mbed-greentea<0.2.0" --upgrade
-```
-
-*Note*: [Greentea v0.1.x](https://github.com/ARMmbed/greentea/tree/master_v0_1_x) is still developed on a ```master_v0_1_x``` branch. We will only apply critical patches to version 0.1.x, no feature development is planned.
-
-### utest support
-
-```utest``` is our preferred test harness which allows you to execute a series of (asynchronous) C++ test cases.  ```greentea-client``` includes as test dependency ```utest``` yotta module. See the ```greentea-client``` [module.json](https://github.com/ARMmbed/greentea-client/blob/master/module.json) ```testDependency``` section:
-```json
-{
-  "testDependencies": {
-    "utest": "^1.10.0"
-  }
-}
-```
-
-### greentea-client support in your module
-
-Currently ```greentea-client``` is in on version ```1.0.x```. Please use a ```^1.0.0``` dependency version in your ```module.json``` file.
-
-Example of ```module.json``` file with ```greentea-client``` as a test dependency:
-```json
-{
-  "testDependencies": {
-    "greentea-client": "^1.0.0",
-    "utest" : "^1.10.0",
-    "unity" : "^2.1.0"
-  }
-}
-```
+* [Greentea](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-greentea)
+* [htrun](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-host-tests)
 
 ## Terms
 
 ### Test suite
-A test suite is a binary containing test cases we execute on hardware. The test suite has a beginning and an end (like your ```main()``` function would. The test suite may pass, fail or be in an error state (for example if test suite times out or there was a serial port connection problem).
-
-### Test case
```

Comment:
Shouldn't we describe what a test case is still? We might have to write a new paragraph for it, rather than talk about minar and utest.